### DISCUSSION TITLE
Sort Actor(person page) Appearances by Release Date/Popularity/

### DISCUSSION THREAD

**Discussion by Philip-Beukes** [seerr-team/seerr#954] [Ideas]
can I please request a new feature, when on the Person Page of the Actor/Acrtress the Appearances are sorted somehow not sure if it is popularity or some score, can we have the option to sort by Release Date/Popularity/TMDB/Title as per the movies Page.

**Comment by up2you1** [comment]
Came on here to ask this. Would be great! Thanks.

**Comment by gajotres** [comment]
I would also like to have this feature, it should be easy to implement as all required information is available. Plus this was a feature that bothered me the most while using old Overseerr.
